src/app/Library/CrudPanel/Traits/Fields.php

```diff
@@ -409,6 +413,17 @@ public function addLoadedFieldType($field)
         return false;
     }
 
+    public function isAnyTypeLoaded($fieldTypes)
```

Comment:
Add doc-block! Check if any of the provided fields in array ['select2','select2_multiple']

Why this ? We still face a problem in crud, and i think this would allows us to solve that. If you have a crud with 2 fields: `select2`, `select2_multiple` we will load twice the `select2.js` because it's used in both fields, but they are different field types. Add ajax selects to the list and you could endup with 4 select2.js in the page. 

ATM this function is only used to include the selects.js (only once for all field types that use it: select2 and select2_multiple)